frontend/app/(tabs)/index.tsx

```diff
@@ -1,6 +1,45 @@
 import { Text, View } from '@/components/Themed';
```

Comment:
These should not be from the `@/components/Themed`. If there is a component that can be imported from the `tamagui` library it should be used. There might be times (possibly in the upcoming weeks) where we would need custom components and those are also going to use `tamagui` components.

Reply:
Removed and replaced with tamagui components